### PR TITLE
Impl. external config of URLs hard-coded to HTTPS

### PR DESCRIPTION
**Pull request by pvannierop**
In order to run and test on a system w/o TLS (e.g., local development system or CI system) some URLs hard-coded to the HTTPS protocol were made configurable.

# Discuss
An alternative would be to add a single value disable_tls that selectively chooses between HTTP and HTTPS protocols. This is done f.i. in the Minio chart:

```
{{ ternary "HTTPS" "HTTP" .Values.tls_enabled | quote }}
```

### CODE REVIEW

**Review by keyvaann — APPROVED** [review]
LGTM

File: charts/management-portal/templates/deployment.yaml [hunk 1]
```diff
@@ -49,6 +49,7 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{ $https := ternary "http" "https" (or .Values.disable_tls (not .Values.ingress.tls)) }}
```
Comment (keyvaann) [inline charts/management-portal/templates/deployment.yaml:52]: Let's put this on top of the file, first few lines

Comment (pvannierop) [reply charts/management-portal/templates/deployment.yaml:52]: Done!

**Review by keyvaann — APPROVED** [review]
LGTM

**Pull request merged**